runtime/environment.yml

```diff
@@ -31,3 +31,6 @@ dependencies:
   - ultralytics=8.1.11
   - tqdm=4.66.2
   - xgboost=2.0.3
+  - openvino=2023.3.0
```

Comment:
@dazmashaly If you remove the other two lines below this line and change the version so it is `openvino=2023.2.0` then it should work.

Do you still need this dependency?

Reply:
@isms yes thank you, I was still trying to solve it.